core/bot/db.php

```diff
@@ -1,13 +1,32 @@
 <?php
+
+require_once('logic/sql_utils.php');
```

Comment:
All the logic stuff is included in `core/bot/requirements.php` that in turn is included before `core/bot/db.php` in index.php, so this is not needed.